cnf-tests/testsuites/e2esuite/security/tuning_sriov.go

```diff
@@ -0,0 +1,90 @@
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/networks"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+
+var sriovclient *sriovtestclient.ClientSet
+
+func init() {
+	sriovclient = sriovtestclient.New("")
+}
+
+var _ = Describe("[sriov] Tuning CNI integration", func() {
+	apiclient := client.New("")
+
+	execute.BeforeAll(func() {
+		err := namespaces.Create(TestNamespace, apiclient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		err := namespaces.CleanPods(TestNamespace, apiclient)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Context("tuningcni over sriov", func() {
+		execute.BeforeAll(func() {
+			if discovery.Enabled() {
+				Skip("Tuned sriov tests disabled for discovery mode")
+			}
+			networks.CleanSriov(sriovclient, TestNamespace)
+			networks.CreateSriovPolicyAndNetwork(sriovclient, namespaces.SRIOVOperator, "test-network", "testresource", networks.SysctlConfig(map[string]string{Sysctl: "1"}))
+		})
+
+		It("pods with sysctl's over sriov interface should start", func() {
+			podDefinition := pods.DefineWithNetworks(TestNamespace, []string{fmt.Sprintf("%s/%s", namespaces.SRIOVOperator, "test-network")})
+			pod, err := client.Client.Pods(TestNamespace).Create(context.Background(), podDefinition, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			err = pods.WaitForCondition(client.Client, pod, corev1.ContainersReady, corev1.ConditionTrue, 1*time.Minute)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("pods with sysctl's on bond over sriov interfaces should start", func() {
+			// NOTE: due to a bond cni bug we need to specify the name of the bond interface in both the cni config and
+			// the multus annotation for the network.
+			bondLinkName := "bond0"
+			bondNetworkAttachmentDefinition, err := networks.NewNetworkAttachmentDefinitionBuilder(TestNamespace, "bond").WithBond(bondLinkName, "net1", "net2").WithTuning(map[string]string{Sysctl: "1"}).Build()
+			Expect(err).ToNot(HaveOccurred())
+			err = client.Client.Create(context.Background(), bondNetworkAttachmentDefinition)
+			Expect(err).ToNot(HaveOccurred())
+
+			podDefinition := pods.DefineWithNetworks(TestNamespace, []string{
+				fmt.Sprintf("%s/%s", namespaces.SRIOVOperator, "test-network"),
+				fmt.Sprintf("%s/%s", namespaces.SRIOVOperator, "test-network"),
+				fmt.Sprintf("%s/%s@%s", TestNamespace, "bond", bondLinkName),
+			})
+			pod, err := client.Client.Pods(TestNamespace).Create(context.Background(), podDefinition, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			err = pods.WaitForCondition(client.Client, pod, corev1.ContainersReady, corev1.ConditionTrue, 1*time.Minute)
+			Expect(err).ToNot(HaveOccurred())
+		})
+	})
+
+	Context("clean SRIOV configuration", func() {
```

Comment:
discovery is disable for this tests so why we need the clean?

Reply:
removed